test/charges/charge_test.exs

```diff
@@ -130,6 +130,15 @@ defmodule PINXS.Charges.ChargeTest do
     end
   end
 
+  test "Search for a charge" do
+    use_cassette("search_charges") do
+      {:ok, retreived_charges} = Charge.search(%{query: "hagrid@hogwarts.wiz"})
```

Comment:
Ha, are these actual responses from some test server?

Reply:
Yeah cool, love the fact it's Harry Potter themed 😂

Reply:
I'm kinda obsessed ... and there's lots of material to work with 😉 🎩 ✨

Reply:
Oh, so _you_ set the "Dragon Eggs" stuff up on Pin? Ha, I thought it was just a "test" response!

Reply:
Guilty 😉